.scalafmt.conf

```diff
@@ -1,8 +1,9 @@
-version = 2.7.5
+version = 3.0.0
 
 style = defaultWithAlign
 
-docstrings                 = JavaDoc
+docstrings.style           = Asterisk
```

Comment:
`JavaDoc` is now `Asterisk` (https://scalameta.org/scalafmt/docs/configuration.html#docstringsstyle--asterisk)